sdks/python/apache_beam/runners/interactive/interactive_runner_test.py

```diff
@@ -147,6 +150,97 @@ def process(self, element):
     ]
     self.assertEqual(actual_reified, expected_reified)
 
+  def test_streaming_wordcount(self):
+    class WordExtractingDoFn(beam.DoFn):
+      def process(self, element):
+        text_line = element.strip()
+        words = text_line.split()
+        return words
+
+    # Add the TestStream so that it can be cached.
+    ib.options.capturable_sources.add(TestStream)
+    ib.options.capture_duration = timedelta(seconds=1)
+
+    p = beam.Pipeline(
+        runner=interactive_runner.InteractiveRunner(),
+        options=StandardOptions(streaming=True))
+
+    data = (
+        p
+        | TestStream()
+            .advance_watermark_to(0)
+            .advance_processing_time(1)
+            .add_elements(['to', 'be', 'or', 'not', 'to', 'be'])
+            .advance_watermark_to(20)
+            .advance_processing_time(1)
+            .add_elements(['to', 'be', 'or', 'not', 'to', 'be'])
+            .advance_watermark_to(40)
+            .advance_processing_time(1)
```

Comment:
Does this not trigger the capture duration?

Reply:
No, because the fake clock is instantiated by the runner, the background caching job only has a handle on the real clock.